packages/agents/sequencer/src/config.ts

```diff
@@ -69,6 +70,7 @@ export const getEnvConfig = (
       configFile.supportedBidVersion ||
       version,
     subgraphPrefix: process.env.SEQ_SUBGRAPH_PREFIX || configJson.subgraphPrefix || configFile.subgraphPrefix,
+    auctionRoundDepth: process.env.AUCTION_ROUND_DEPTH || configJson.auctionRoundDepth || DEFAULT_AUCTION_ROUND_DEPTH,
```

Comment:
as discussed in discord -- will need some v
<img width="1216" alt="Screen Shot 2022-06-01 at 8 53 05 PM" src="https://user-images.githubusercontent.com/12769011/171542997-2e3aed9e-ad60-406d-99d9-b5df19ac283f.png">
alidation around the maximum routers you include with this bid path. context:

Reply:
@wanglonghong's reply to this:
![image](https://user-images.githubusercontent.com/37907740/171543711-63fb12b5-bec6-4a06-9d64-1de8710fafe9.png)

Reply:
resolve in https://github.com/connext/nxtp/issues/1343